powdr/src/lib.rs

```diff
@@ -219,8 +221,8 @@ pub fn run(pipeline: &mut Pipeline<GoldilocksField>) {
     );
 
     let duration = start.elapsed();
-    log::info!("Fast executor took: {:?}", duration);
-    log::info!("Trace length: {}", trace.len);
+    println!("Fast executor took: {duration:?}");
```

Comment:
Debug print `{:?}` isn't really meant for end user printing. But in this case, I don't think there is a better option aside from formatting manually.